modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -47,6 +48,9 @@ def unwrap_partitions(api_layer_object, axis=None, bind_ip=False):
             f"Only API Layer objects may be passed in here, got {type(api_layer_object)} instead."
         )
 
+    if bind_ip and not EnablePartitionIPs.get():
+        raise ValueError("Passed `bind_ip=True` but partition IPs API was not enabled.")
```

Comment:
As I understand, this exception will raised now when user will work with Modin XGBoost, because `bind_ip=True` already used in XGB. Could you add mentioning of partition IPs enabling in Modin XGBoost docpage?

Reply:
not actual.